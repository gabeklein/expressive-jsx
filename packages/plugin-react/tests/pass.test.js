const test = require("./_adapter");

test("collapse style-only expression", `
  const props = do {
    color: red;
  }
`);