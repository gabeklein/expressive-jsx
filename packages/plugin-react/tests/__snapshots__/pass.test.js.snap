// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`collapse style-only expression 1`] = `
"const props = {
  \\"className\\": \\"props_67v\\"
};
Styles.include(\`
	.props_67v { 
		color: red;
	}
\`);"
`;

exports[`collapse style-only expression 2`] = `
"const props = {
  \\"className\\": \\"props_67v\\"
};
Styles.include(\`
	.props_67v { 
		color: red;
	}
\`);"
`;
